lib/internal/errors.js

```diff
@@ -101,6 +101,8 @@ module.exports = exports = {
 // Note: Please try to keep these in alphabetical order
 E('ERR_ARG_NOT_ITERABLE', '%s must be iterable');
 E('ERR_ASSERTION', '%s');
+E('ERR_ASYNC_CALLBACK', (name) => `${name} must be a function`);
```

Comment:
Could you add perfunctory tests in `test/parallel/test-internal-errors.js` as per https://github.com/nodejs/node/blob/master/doc/guides/using-internal-errors.md#testing-new-errors